front/src/app/todo-list/components/active-task-list/active-task-list.component.scss

```diff
@@ -20,6 +20,9 @@
   }
   &__new-task-input {
     max-height: 70px;
+    max-width: 300px;
+    width: 300px;
+    min-width: 200px;

```

Comment:
having width here feels weird to me - the app is split into left and right working areas, the right working area shouldn't be too big (makes sense to me) but its max-width has to be defined once on some higher level and that's it. All the inner elements should inherit its width in one or another way without using concrete amount of pixels